src/elements/content-sidebar/SidebarRouter.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @flow
+ * @file Sidebar Router Component
+ * @author Box
+ */
+import * as React from 'react';
+import { MemoryRouter, Router } from 'react-router-dom';
+import type { RouterHistory } from 'react-router-dom';
+
+type Props = {
+    children: React.Node,
+    history?: RouterHistory,
+};
+
+const SidebarRouter = ({ children, history, ...rest }: Props) => {
```

Comment:
Will this eventually be at the ContentPreview level instead of just the Sidebar?

Reply:
That's the ultimate goal, but it'll require top-level routes in our host application.